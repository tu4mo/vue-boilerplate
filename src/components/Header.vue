<template>
  <header>
    <h1>Vue Boilerplate</h1>
    <nav>
      <router-link to="/" exact>Home</router-link>
      <router-link to="/page">Page</router-link>
    </nav>
  </header>
</template>

<style lang="scss" scoped>
header {
  align-items: center;
  background-color: purple;
  color: #fff;
  display: flex;
  padding: 1rem;
}

h1 {
  margin-right: 1rem;
}

a {
  color: #fff;
  text-decoration: none;
  padding: 1rem;
}

.router-link-active {
  background-color: rgba(255, 255, 255, 0.1);
}
</style>
