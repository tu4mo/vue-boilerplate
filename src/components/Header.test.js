import { shallow } from '@vue/test-utils'
import Header from './Header.vue'

describe('Header', () => {
  it('should render correctly', () => {
    const component = shallow(Header, {
      stubs: ['router-link']
    })
    expect(component.element).toMatchSnapshot()
  })
})
