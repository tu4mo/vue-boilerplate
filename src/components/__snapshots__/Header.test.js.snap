// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly 1`] = `
<header>
  <h1>
    Vue Boilerplate
  </h1>
   
  <nav>
    <!---->
     
    <!---->
  </nav>
</header>
`;
